torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -1031,10 +1023,10 @@ void RemoveInplaceOpsForONNX(
   MutationRemover mr(graph);
   ImplicitCastForBinaryInplaceOps(graph->block());
   PrepareForRemoveMutations(mr, graph->block());
-  RemoveTensorMutation(graph);
-  RemoveListMutation(graph);
   if (model)
     RegisterInplaceOpAsBlockOutputs(mr, model, graph);
+  RemoveTensorMutation(graph);
```

Comment:
why moving after register as block outputs?

Reply:
Registration pass would need to scan for inplace ops, bug inplace ops will be removed after RemoveMutation pass.